app/main.py

```diff
@@ -1,57 +1,58 @@
+from __future__ import annotations
+
+
 class Distance:
-    def __init__(self, km):
+    def __init__(self, km: int) -> None:
         self.km = km
 
-    def __str__(self):
+    def __str__(self) -> str:
         return f"Distance: {self.km} kilometers."
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"Distance(km={self.km})"
 
-    def __add__(self, other):
+    def __add__(self, other) -> Distance:
         if isinstance(other, Distance):
             return Distance(self.km + other.km)
         return Distance(self.km + other)
 
-    def __iadd__(self, other):
+    def __iadd__(self, other) -> Distance:
         if isinstance(other, Distance):
             self.km += other.km
         else:
             self.km += other
         return self
 
-    def __mul__(self, other):
-        if isinstance(other, Distance):
-            return Distance(self.km * other.km)
+    def __mul__(self, other) -> Distance:
         return Distance(self.km * other)
 
-    def __truediv__(self, other):
+    def __truediv__(self, other) -> Distance:
         return Distance(round((self.km / other), 2))
 
-    def __lt__(self, other):
+    def __lt__(self, other) -> bool:
         if isinstance(other, Distance):
             return self.km < other.km
         return self.km < other
 
-    def __gt__(self, other):
+    def __gt__(self, other) -> bool:
         if isinstance(other, Distance):
             return self.km > other.km
         return self.km > other
 
-    def __eq__(self, other):
+    def __eq__(self, other) -> bool:
         if isinstance(other, Distance):
             return self.km == other.km
         return self.km == other
 
-    def __le__(self, other):
+    def __le__(self, other) -> bool:
         if isinstance(other, Distance):
             return self.km <= other.km
         return self.km <= other
 
-    def __ge__(self, other):
+    def __ge__(self, other) -> bool:
         if isinstance(other, Distance):
             return self.km >= other.km
         return self.km >= other
```

Comment:
you skipped my previous comment:
> try to rewrite these two methods in one line each. Use previously created magic methods
Hint: <= == not >

tag me on your slack channel if you don't understand what should you do

Reply:
I didn't notice, sorry. I've already changed it